### ISSUE TITLE
Migrate content from civicrm-core/tests/README.md

### ISSUE THREAD

**Issue by seanmadsen** [civicrm/civicrm-dev-docs#63]
There is some content here related to testing:

https://github.com/civicrm/civicrm-core/blob/master/tests/README.md

I would like to move this content into the Dev Guide and delete the file from civicrm-core. Is that okay @totten ?

**Comment by totten** [comment]
Yeah, we don't want to end up with multiple test guides, so it makes sense to migrate that when writing a test chapter.

The main thing is discovery - some people learn about a codebase by going to Github or the filesystem and looking around. A README.md is more discoverable for them. And keeping the README in `civicrm-core` means that you get details which are in-sync with the code. However, we've previously discussed moving the dev docs back to core repo; when that happens, this concern would be resolved.

For the interim, I'd say... go ahead and migrate it. But instead of deleting the file, leave a small README.md with a link to the dev doc.

**Comment by seanmadsen** [comment]
Nice plan @totten

**Comment by totten** [comment]
More concrete list of steps/suggestions/todos:

 * There's a table in [tests/README.md](https://github.com/civicrm/civicrm-core/blob/4.7.23/tests/README.md) that summarizes the technical differences between each of the PHPUnit suites.
   * *Action*: Migrate that to https://docs.civicrm.org/dev/en/latest/testing/phpunit/
 * There are two places that talk about setup (`cv vars:fill`, etal). Specifically, https://docs.civicrm.org/dev/en/latest/testing/phpunit/ and https://github.com/civicrm/civicrm-core/blob/4.7.23/tests/README.md
   * These are most prominently required for `phpunit`, but they can be used by any test-runner.
   * *Action*: Move these into https://docs.civicrm.org/dev/en/latest/testing/setup/ -- include both (a) the nicer walk-through (from `civicrm-dev-docs`) and (b) the more detailed comments about `ADMIN_USER`, `ADMIN_PASS`, etal (from `civicrm-core`).
 * We don't want the `README.md` to be redundant.
   * *Action*: Update `README.md` to be a hyperlink to the Dev Guide

**Comment by seamuslee001** [comment]
@totten are you folk going to be working on this or needing someone else to start on the docs?
